pkg/cmd/urfave/urfave.go

```diff
@@ -105,10 +105,15 @@ func GetTraceeRunner(c *cli.Context, version string) (cmd.Runner, error) {
 	// Check kernel lockdown
 
 	lockdown, err := helpers.Lockdown()
-	if err == nil && lockdown == helpers.CONFIDENTIALITY {
+	if err != nil {
+		logger.Error("kernel lockdown", "error", err)
+	}
+
+	if lockdown == helpers.CONFIDENTIALITY {
```

Comment:
To be on the safe side, better check error is nil

Reply:
Oh! That ia true, because we are not returning!